packages/checkout-ui-extensions/src/components/BlockLayout/examples/basic-blockLayout.example.ts

```diff
@@ -0,0 +1,16 @@
+import {extend, BlockLayout, View} from '@shopify/checkout-ui-extensions';
+
+extend('Checkout::Feature::Render', (root) => {
```

Comment:
What's the difference between `'Checkout::Dynamic::Render'` and `'Checkout::Feature::Render'` - should it just be `'Checkout::Dynamic::Render'` for consistency?